.github/workflows/call-refresh-doc.yml

```diff
@@ -0,0 +1,13 @@
+name: call-refresh-doc
+on:
+  push:
+    branches:
+      - hannah-github-actions
+
+jobs:
+  call-workflow-passing-data:
+    uses: onflow/cadence/.github/workflows/refresh-doc.yml@hannah-github-actions
+    with:
+      username: codingone21
```

Comment:
Is this for testing purposes? Should this go into `master`?

Reply:
This should eventually be replaced to master in a later PR, but since we are still in testing stages, I wanted to keep the changes in my branch! I don't think we would need this on master yet.

Reply:
I ended using @master for the yaml file, since I do most of the testing locally anyways. Didn't want to complicate it too much by making different versions of the same branch 😅